hack/mkdocs-copy-geps.py

```diff
@@ -15,10 +15,109 @@
 import shutil
 import logging
 from mkdocs import plugins
+import yaml
```

Comment:
Nit: I think this file should probably be renamed to represent the broader scope of the script.

Reply:
im thinking of putting this conformance report generation into it's own `make-implementation-table.py` and have it as it's own hook. Thoughts??